elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigSchemaValidator.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import com.github.fge.jsonschema.core.report.ProcessingReport;
+import com.github.fge.jsonschema.main.JsonSchema;
+import com.github.fge.jsonschema.main.JsonSchemaFactory;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+
+@Slf4j
+public class DynamicConfigSchemaValidator {
+
+    private static final String TABLE_SCHEMA = "/elideTableSchema.json";
+    private static final String SECURITY_SCHEMA = "/elideSecuritySchema.json";
+    private static final String VARIABLE_SCHEMA = "/elideVariableSchema.json";
+
+    private static final JsonSchemaFactory FACTORY = JsonSchemaFactory.byDefault();
+    private static JsonSchema tableSchema;
+    private static JsonSchema securitySchema;
+    private static JsonSchema variableSchema;
+
+    static {
+        tableSchema = loadSchema(TABLE_SCHEMA);
+        securitySchema = loadSchema(SECURITY_SCHEMA);
+        variableSchema = loadSchema(VARIABLE_SCHEMA);
+    }
+
+    /**
+     *  Verify config against schema.
+     * @param configType
+     * @param jsonConfig
+     * @return whether config is valid
+     * @throws IOException
+     * @throws ProcessingException
+     */
+    public static boolean verifySchema(String configType, String jsonConfig) throws IOException, ProcessingException {
```

Comment:
Is there a reason all of this has to be static?  It will be easier to mock if it were not.